### PR TITLE
[16.0][MIG] partner_category_security: Migration to 16.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 16.0

Please @pilarvargas-tecnativa can you review it?

@Tecnativa TT47846

### CODE REVIEW

File: base_partner_sequence/README.rst [hunk 1]
```diff
@@ -7,7 +7,11 @@ Add a sequence on customers' code
    !! This file is generated by oca-gen-addon-readme !!
    !! changes will be overwritten.                   !!
    !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+<<<<<<< HEAD
    !! source digest: sha256:f9cc8be18f0a5732ba23036f7f3535e25835869d1a76d77d673d5c3943029886
+=======
+   !! source digest: sha256:05da69d5fc41642ad7e60a53ab9b42bbc8575678cfc35671ee34935911a81b9b
+>>>>>>> 62915c78 ([UPD] README.rst)
```
Comment (chienandalu) [inline base_partner_sequence/README.rst]: You've got merge conflicts :point_up:

Comment (victoralmau) [reply base_partner_sequence/README.rst]: Fixed.

**Review by chienandalu — APPROVED** [review]

**Review by pilarvargas-tecnativa — APPROVED** [review]

**Review by HaraldPanten — APPROVED** [review]
LGTM functional review in runboat

**Comment by HaraldPanten** [comment]
> /ocabot migration partner_category_security

We thought the same 😂

**Pull request merged**